fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/Type.java

```diff
@@ -435,6 +442,19 @@ public static TypeCode fromProtobufType(@Nonnull final Descriptors.FieldDescript
      * Special {@link Type} that is undefined.
      */
     class Any implements Type {
+        /**
+         * Memoized hash function.
+         */
+        @Nonnull
+        private final Supplier<Integer> memoizedHashFunction;
+
+        /**
+         * Constructs a new instance of {@link Any} type.
+         */
+        public Any() {
+            this.memoizedHashFunction = Suppliers.memoize(() -> Objects.hash(getTypeCode().hashCode(), isNullable()));
```

Comment:
so normally we use the pattern `xxxSupplier = Suppliers.memoize(this::computeXXX)` and then

`computeHashCode()` would do the `Objects.hash(...)`.

Reply:
done.